conf/defaults.yaml

```diff
@@ -162,6 +168,13 @@ supervisor.memory.capacity.mb: 4096.0
 # for single threaded bolts
 supervisor.cpu.capacity: 400.0
 
+#Supervisor thrift config
+supervisor.thrift.port: 6627
```

Comment:
We often setup test cluster in a node, hence two ports (nimbus/supervisor) should be different.

Reply:
done